da/test/da_test.go

```diff
@@ -140,6 +149,38 @@ func TestRetrieve(t *testing.T) {
 	}
 }
 
+func startMockGRPCServ() *grpc.Server {
```

Comment:
we should leave `startMockGRPCServ` and `startMockCelestiaNodeServer` in `da/test/da_test_helpers.go`